boards/mcb2388/Makefile.features

```diff
@@ -0,0 +1,9 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
+FEATURES_PROVIDED += periph_pwm
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+include $(RIOTBOARD)/common/msba2/Makefile.features
```

Comment:
haven't checked but doesn't the common msba2 board provide some of the features above ?

Reply:
In currently only contains:

```
CPU = lpc2387
```

I bet a lot more common code could be moved there. But maybe it is better to leave this as its own PR.

Reply:
It makes more sense to properly define the CPU here.
I removed the useless include.